sdk/appconfiguration/app-configuration/src/internal/tracingHelpers.ts

```diff
@@ -90,7 +91,7 @@ export class Spanner<TClient> {
       ...options,
       spanOptions: {
         ...spanOptions,
-        parent: span,
+        parent: span.context(),
```

Comment:
I'd expect the parent of a span to also be of type span... this would be one of the things the user can pass in right?
We are changing that now, or was that never the case?

Reply:
OpenTelemetry lets you pass either a `Span` or a `SpanContext`. This is mostly for convenience as it is not hard to get the context. 

However, since `Span` is a **much** bigger surface area than context, the proposal is to not support direct passing of spans in our options in order to avoid having to risk breaking changes in `Span`.

Reply:
But yes, this is a breaking change.